pkg/domain/infra/abi/images.go

```diff
@@ -722,10 +722,6 @@ func (ir *ImageEngine) Sign(ctx context.Context, names []string, options entitie
 		if err != nil {
 			return nil, errors.Wrapf(err, "error getting image source")
 		}
-		err = rawSource.Close()
```

Comment:
There is a chance at rawsouce not being closed.  I agree with Miloslav, to move this to a function and 
defer rawSource.Close()